src/E1_coldflow/E1_coldflow.cpp

```diff
@@ -68,6 +68,7 @@ void setup() {
   if (!res) {
     packet = make_packet(101, true);
     RFSerial.println(packet);
+    incrementPacketCounter();
```

Comment:
Notice how to increment is always called along with make packet? I think we should just put the increment function inside `make_packet`